main.go

```diff
@@ -28,6 +29,9 @@ func main() {
 		context.Rest.SetRoutes(c.Routes())
 	}
 
+	// Store self information in the store
+	peer.AddSelfDetails()
```

Comment:
well, later in some use case like (AddSelfVolumeDetails or something else). by making it peer or node its making more meaningful.